userspace/falco/app/actions/init_inspectors.cpp

```diff
@@ -187,6 +187,23 @@ falco::app::run_result falco::app::actions::init_inspectors(falco::app::state& s
 		{
 			return run_result::fatal(err);
 		}
+
+		// in live mode, each inspector should have registered at most two event sources:
+		// the "syscall" on, loaded at default at index 0, and optionally another
+		// one defined by a plugin, at index 1
```

Comment:
Just a comment: oh max one plugin is a limitations we may wanna consider addressing one day if we plan to extend the plugins functionally beyond what is possible today :)